src/BaGet.Aws/AwsApplicationExtensions.cs

```diff
@@ -31,11 +31,24 @@ public static BaGetApplication AddAwsS3Storage(this BaGetApplication app)
             {
                 var options = provider.GetRequiredService<IOptions<S3StorageOptions>>().Value;
 
+                if (options.ForcePathStyle)
+                {
+                    // Required to support presigned urls generation for MiniO configured to us-east-1 region
+                    AWSConfigsS3.UseSignatureVersion4 = true; 
+                }
+
                 var config = new AmazonS3Config
                 {
-                    RegionEndpoint = RegionEndpoint.GetBySystemName(options.Region)
+                    RegionEndpoint = RegionEndpoint.GetBySystemName(options.Region),
```

Comment:
From my understanding the region can be set using the service URL. If so should we follow @DTeuchert's pattern in https://github.com/loic-sharma/BaGet/pull/272:

```suggestion
                    RegionEndpoint = (options.Region != null)
                        ? RegionEndpoint.GetBySystemName(options.Region)
                        : null,
                    ServiceURL = options.ServiceUrl
```

Reply:
I think the AWS SDK requires a region independent from service url